sdk/appconfiguration/app-configuration/samples/react-sample/src/components/Page.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT license.
+*/
+
+import React from "react";
+import { useEffect, useState } from "react";
+import {
+  Navbar,
+  Nav,
+  NavDropdown,
+  Jumbotron,
+  Alert,
+  Row,
+  Col,
+  Container
+} from "react-bootstrap";
+import { getEnvironmentVariable } from "../utils";
+import {
+  AppConfigurationClient,
+  featureFlagPrefix,
+  isFeatureFlag,
+  isFeatureFlagClientFilter
+} from "@azure/app-configuration";
+
+let client: AppConfigurationClient;
+
+const featureFlag1 = {
+  id: "react-app-feature-1",
+  description: "",
+  enabled: false,
+  conditions: {
+    client_filters: []
+  }
+};
+const featureFlag2 = {
+  id: "react-app-feature-2",
+  description: "",
+  enabled: true,
+  conditions: {
+    client_filters: [
+      {
+        name: "Microsoft.TimeWindow",
+        parameters: {
+          Start: "Mon, 12 Apr 2021 07:00:00 GMT",
+          End: "Wed, 14 Apr 2021 07:00:00 GMT"
+        }
+      }
+    ]
+  }
+};
+
+export default function Page(): JSX.Element {
+  const feature1 = "react-app-feature-1";
+  const feature2 = "react-app-feature-2";
+  const [feature1ClassName, setFeature1ClassName] = useState<string>("feature-button-hidden");
+  const [feature2ClassName, setFeature2ClassName] = useState<string>("feature-button-hidden");
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const getFeatureFlags = async (keys: string[]): Promise<void> => {
+    const [setting1, setting2] = await Promise.all(
+      keys.map((key: string) =>
+        client.getConfigurationSetting({
+          key: featureFlagPrefix + key
+        })
+      )
+    );
+    if (isFeatureFlag(setting1)) {
+      console.log(`${setting1.key} is enabled : ${setting1.enabled}`, setting1);
```

Comment:
Is this (and on line 73) just debugging logs that should be deleted or will the user find them useful when running the sample?

Reply:
Just some debug logs for the user.